src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write your code here
+function successHandler(text) {
+  const message = document.createElement('div');
+
+  message.className = 'message';
+  message.textContent = text;
+  document.body.append(message);
+}
+
+function errorHandler() {
```

Comment:
```suggestion
function errorHandler(error) {
```
and then `message.textContent = error.message`. I meant you can read info from error
Just FYI